common/src/main/java/com/wynntils/gui/screens/maps/MainMapScreen.java

```diff
@@ -91,13 +91,7 @@ protected void init() {
                                 .append(Component.translatable("screens.wynntils.map.help.description6")),
                         Component.literal("- ")
                                 .withStyle(ChatFormatting.GRAY)
-                                .append(Component.translatable("screens.wynntils.map.help.description7")),
```

Comment:
And these are not unused -- they merele become so since you remove functionality below..!?!